### PR TITLE
Re-add AWS logs release note 

### PR DESCRIPTION
**Pull request by jpipkin1**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request re-adds a release note originally added by PR #3339 (but reverted by PR #3341). The release note will be published here: 
https://help.sumologic.com/release-notes-service/2023/11/28/observability/

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

**Comment by jpipkin1** [comment]
@kimsauce, could you review this PR?

File: blog-service/2023-11-28-observability.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+We are excited to announce several improvements to our AWS Observability solution.  
+
+* **AWS Observability Hub**. A landing page to access all the capabilities of the Sumo Logic AWS Observability solution, including the out-of-the-box dashboards, alerts, and performance benchmark reports, all from a single place.
+* **Log Searches for Troubleshooting**. Quickly troubleshoot issues within your AWS environment with pre-baked log searches. No need to learn the search query syntax. These new searches can be accessed from the new hub page.
+
+To learn more about the AWS Observability solution and how to set it up, check out our documentation [here](/docs/observability/aws/). 
+
+:::note
+To access these new capabilities, you just have to install the AWS Observability solution. If you have one already installed, congratulations! You'll gain access to these new features automatically. 
+:::
```
Comment (kimsauce) [inline blog-service/2023-11-28-observability.md:26]: Because these don't render in rss feeds, I usually use just plain text "NOTE: ..."

**Review by kimsauce — APPROVED** [review]

File: blog-service/2023-11-28-observability.md (hunk 1 above)
Comment (jpipkin1) [reply blog-service/2023-11-28-observability.md:26]: Oh, I see! I'll change it back.

Comment (jpipkin1) [inline blog-service/2023-11-28-observability.md:26]:
```suggestion
NOTE: To access these new capabilities, you just have to install the AWS Observability solution. If you have one already installed, congratulations! You'll gain access to these new features automatically. 
```

**Pull request merged**